less/about/autofill.less

```diff
@@ -1,4 +1,8 @@
-@import "./common.less";
```

Comment:
Removed as it is required on autofill.js See: https://github.com/brave/browser-laptop/pull/10881/files#diff-f3189199f630a8432fae752aa453b2ccR15